packages/server/src/@trpc/server/index.ts

```diff
@@ -32,6 +32,8 @@ export {
   type RouterRecord as TRPCRouterRecord,
   type AnySubscriptionProcedure as AnyTRPCSubscriptionProcedure,
   type ProcedureOptions as TRPCProcedureOptions,
+  type CreateContextCallback,
+  type WrapCreateContext,
```

Comment:
Do these need to be exported?

if so, smurf them 👀

Reply:
Yes they do, I was getting the usual portability error on the frontend